src/web/modules/transactions/transactions.http.ts

```diff
@@ -77,3 +81,59 @@ export async function show(req: Request, res: Response, next: NextFunction): Pro
     next(error)
   }
 }
+
+export async function statistics(req: Request, res: Response, next: NextFunction): Promise<void> {
+  try {
+    let account
+    const accountId = req.query.account
+
+    if (req.query.account) {
+      account = await AdminUsers.gatewayAccountServices(accountId)
+    }
+
+    const selectedPeriod: string = req.query.period || 'Today'
+
+    const dateInUTC = moment().utc()
+    let fromDate: string;
+    let toDate: string;
+    switch(selectedPeriod) {
```

Comment:
A neat way of doing this switch statement in a more functional way would be to use an object map, it would: 
* remove any branching paths, variables will only be defined once according to the map so there's no chance it's in an unknown state
* reduce lines of code

This converting a token (string) into a from date and a to date would be a great thing to unit test to protect against regressions. 

Final suggestion, removing spaces and special characters from the key would help remove places where things could go wrong with string matching, this is especially important as it will be converted from a URL and back again at least once. 

The suggested way might look roughly like: 
```js 
const periodKeyMap = { 
  today: 'day',
  thisWeek: 'week', 
  thisMonth: 'month'
}
const selectedPeriod = req.query.period || 'today'
const momentKey = periodKeyMap[selectedPeriod]

const fromDate = Moment.utc().startOf(momentKey).format()
const toDate = Moment.utc().endOf(momentKey).format()
```

Reply:
Done